ls1intum/Artemis
# ``General``: Improve instructor course overview style

### Checklist
#### General


#### Client

### Motivation and Context


As pointed out by @krusche , the instructor course overview page does currently have a few styling and responsiveness issues:
- Participation score bar moved into the next "line", overlaying the following exercise in certain viewport widths (1251-1271, 1501-1549, ~1720).
- Overuse of gray borders with weird margins in a few cases
- Text overflows on mobile screens
- Too many footer buttons shown on mobile screens

### Description
This PR will attempt to fix these issues by suggesting a more polished appearance.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 to 10 active exercises in a course

1. Log in to Artemis
2. Navigate to Course Administration
3. Inspect style; try responsiveness using "test screen sizes" feature of your browsers and check for any errors

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage

### Screenshots
Before
![grafik](https://user-images.githubusercontent.com/23171488/142689461-68087da7-0fba-406e-b783-7a596339827b.png)

![grafik](https://user-images.githubusercontent.com/23171488/142683920-d4be4fb5-661c-43c7-827d-b3aa7b343ec7.png) ![grafik](https://user-images.githubusercontent.com/23171488/142683934-ba073849-fa76-4f34-a8d3-b92ebb036cee.png)

After:

https://user-images.githubusercontent.com/23171488/142689397-42d2caf9-0c57-4461-ab75-4e4556255e85.mov
